addon-test-support/mock-provider/index.js

```diff
@@ -46,9 +46,10 @@ export default class MockProvider {
    * @method addState
    * @param {string} name
    * @param {object} [params]
+   * @return {number}
    */
   addState(name, params) {
-    this.interaction.addProviderState(name, params);
+    return this.interaction.addProviderState(name, params);
```

Comment:
I was not sure how to proceed here. I am returning whatever is returned by addProviderState.